pydocxresizeimages/util/uri.py

```diff
@@ -35,10 +35,10 @@ def sanitize_filename(filename):
     by a dash (-) to make the image unique for round-tripping. In an effort to
     prevent a bunch of timestamps preceding the image name (in the event a
     document is round-tripped several times), strip off the timestamp
-    and dash. When images come from docx they are always `image\d+`. We only
+    and dash. When images come from docx they are always `imageN`. We only
```

Comment:
The "noqa" on line 41 is supposed to ignore this whole text but the behavior changed at some point. Rather than chasing it down, I'm punting and just changing the text.